### ISSUE TITLE
GSUB 'vrt2' feature is not a superset of the 'vert' feature.

### ISSUE THREAD

**Issue by trueroad** [adobe-fonts/source-han-sans#282]
### Prerequisites

* [x] Did you thoroughly search the [open](https://github.com/adobe-fonts/source-han-sans/issues) and [closed](https://github.com/adobe-fonts/source-han-sans/issues?q=is%3Aissue+is%3Aclosed) issues to avoid reporting a duplicate issue?
* [x] Did you go through the [official font readme file](https://github.com/adobe-fonts/source-han-sans/raw/release/SourceHanSansReadMe.pdf) to better understand the scope of the project, to include the *Known Issues* section at the very end?

### Description

The entry for Tag: 'vrt2' feature in the OpenType specification version 1.8.4 says "Note that these are a superset of the glyphs covered in the 'vert' table."
https://docs.microsoft.com/en-us/typography/opentype/spec/features_uz#tag-vrt2

However, if I understand correctly, in Source Han Sans and Source Han Serif, some glyphs (e.g. U+FF1A 'FULLWIDTH COLON' in Japanese) are covered by the 'vert' feature, but not by the 'vrt2' feature.
This means that 'vrt2' is not a superset of 'vert', but a subset.

I would like 'vrt2' to be a superset containing all the glyphs covered by 'vert' as the specification says.

**Comment by punchcutter** [comment]
The OpenType spec appears to be incorrect and it should say the opposite. A good explanation about `vert` and `vrt2` can be found in this discussion: https://forum.glyphsapp.com/t/automatic-feature-code-must-include-vert-in-vrt2/12297/11

**Comment by punchcutter** [comment]
As`vrt2` is "effectively deprecated" as explained [here](https://forum.glyphsapp.com/t/automatic-feature-code-must-include-vert-in-vrt2/12297/11 ) it is only included for Microsoft apps.

**Comment by trueroad** [comment]
Thank you for the link to an interesting discussion.
Do you mean the following?

* The fact that `vrt2` is a subset of `vert` does not satisfy the Open Type specification.
* However, `vrt2` is "effectively deprecated," so leave it as the subset.

If yes, when we use Source Han fonts with the Microsoft apps, we will be in trouble because we cannot use the vertical glyph of U+FF1A.

**Comment by punchcutter** [comment]
Yes, for one thing the OpenType spec should probably be updated at some point, but the reality is still a mix of different applications and systems handling vertical setting in different ways. As stated in the link above and also in the [ReadMe](https://github.com/adobe-fonts/source-han-sans/blob/release/SourceHanSansReadMe.pdf) the `vrt2` feature is effectively deprecated in favor of [#UAX50](http://www.unicode.org/reports/tr50/tr50-24.html). 

I just checked Microsoft Word on both Mac and Windows and U+FF1A uses the correct vertical glyph so it appears to be using `vert`.

**Comment by bogytech** [comment]
Isn't it better to have  one unified GSUB table for each language? This will increase the size of the font, but the Source Han font is large enough that it won't make much difference.
